tests/api.test.ts

```diff
@@ -422,10 +414,10 @@ describe('conditional mocking', () => {
     it('mocks when matches string', async () => {
       const response = 'blah';
       const response2 = 'blah2';
-      fetch.doMockOnceIf('http://foo/', response).doMockOnceIf('http://foo2/', response2);
+      fetch.mockResponseOnceIf('http://foo/', response).mockResponseOnceIf('http://foo2/', response2);
```

Comment:
Question: why did this change?

Reply:
Resolved, I forgot to add the additional optional arguments. Restored the test to the original implementation.